tests/jq_fuzz_execute.cpp

```diff
@@ -28,8 +28,11 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t *data, size_t size) {
         }
         jv_free(next);
       }
-
-      // Do not free "input" as this is handled by jq_teardown.
+      else {
```

Comment:
nit pick: `} else {` to follow style in other files?

Reply:
Ah yes, thanks! Done!